android/kotlin/v2/AmpliApp/app/src/main/java/com/amplitude/ampli/Ampli.kt

```diff
@@ -0,0 +1,822 @@
+// Tracking Plan Version: 0
+// Build: 1.0.0
+// Runtime: android:kotlin-ampli-v2
+//
+// [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+//
+// [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/kotlin-ampli-v2)
+//
+
+package com.amplitude.ampli
+
+import com.amplitude.android.Amplitude
+import com.amplitude.android.Configuration
+import com.amplitude.android.events.BaseEvent
+import com.amplitude.android.events.EventOptions
+import com.amplitude.android.events.Plan
+
+enum class EventType(val value: String) {
+    Identify("\$identify"),
+    GroupIdentify("\$groupidentify"),
+}
+
+class LoadOptions(
+    val environment: Ampli.Environment? = null,
+    val disabled: Boolean? = null,
+    val client: LoadClientOptions? = null
+)
+
+class LoadClientOptions(
+    val apiKey: String? = null,
+    val instance: Amplitude? = null,
+    val plan: Plan? = null
+)
+
+class Identify private constructor() : BaseEvent() {
+    /**
+     * Identify
+     *
+     * [View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest/Identify)
+     *
+     * Identify properties.
+     *
+     * @param requiredNumber Description for identify requiredNumber
+     * @param optionalArray Description for identify optionalArray
+     */
+    constructor(
+        requiredNumber: Double,
+        optionalArray: Array<String>? = null
+    ) : this() {
+        this.eventType = EventType.GroupIdentify.value
+        this.eventProperties =
+            mapOf(
+                *(if (optionalArray != null) arrayOf("optionalArray" to optionalArray) else arrayOf()),
+                "requiredNumber" to requiredNumber
+            ).toMutableMap()
+    }
+  }
+
+class Group private constructor() : BaseEvent() {
+    /**
+     * Group
+     *
+     * [View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest/Group)
+     *
+     * Group properties.
+     *
+     * @param requiredBoolean Description for group requiredBoolean
+     * @param optionalString Description for group optionalString
+     */
+    constructor(
+        requiredBoolean: Boolean,
+        optionalString: String? = null
+    ) : this() {
+        this.eventType = EventType.Identify.value
+        this.eventProperties =
+            mapOf(
+                *(if (optionalString != null) arrayOf("optionalString" to optionalString) else arrayOf()),
+                "requiredBoolean" to requiredBoolean
+            ).toMutableMap()
+    }
+  }
+class EventMaxIntForTest private constructor() : BaseEvent() {
+    /**
+     * EventMaxIntForTest
+     *
+     * [View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest/EventMaxIntForTest)
+     *
+     * Event to test schema validation
+     *
+     * Owner: a94a8fe5ccb19ba61c4c0873d391e987982fbbd3 06b69c130b0036944180e653185c74f600c6fe43
```

Comment:
Curious about the owner info here, are the two ids the real information?

Reply:
I applied the local codege changes in local. For local env, instead shows the workspace, it will show ids. 
I can replace this with `Test Codegen`